.ai/pages/node-infrastructure-run-a-node-parachain-rpc.md

```diff
@@ -0,0 +1,546 @@
+
+!!! note
+
+    The parameters and configurations in this guide are provided as illustrative examples. You may need to modify them according to your specific environment, hardware capabilities, and network conditions.
+
+## Obtaining a Chain Specification
+
+To run an RPC node for any parachain, you need its **chain specification file**. This JSON file defines the network parameters, genesis state, and bootnodes.
+
+### System Parachains
+
+System parachain chain specs are available from multiple sources:
+
+**Option 1: Chainspec Collection (Recommended)**
+
+Visit the [Chainspec Collection](https://paritytech.github.io/chainspecs/) to download official chain specifications.
+
+**Option 2: Polkadot SDK Repository**
+
+Download directly from the Polkadot SDK repository:
+
+```bash
+# Example for People Chain
+curl -L https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/cumulus/parachains/chain-specs/people-polkadot.json -o chain-spec.json
+```
+
+| System Parachain | Para ID | Chain Spec File | Snapshot Path |
+|------------------|---------|-----------------|---------------|
+| **Bridge Hub** | 1002 | `bridge-hub-polkadot.json` | `polkadot-bridge-hub-rocksdb-archive` |
+| **People Chain** | 1004 | `people-polkadot.json` | `polkadot-people-rocksdb-archive` |
+| **Coretime Chain** | 1005 | `coretime-polkadot.json` | `polkadot-coretime-rocksdb-archive` |
+
+### Other Parachains
+
+For non-system parachains, check the parachain's documentation for official chain specification files.
+
+!!! note
+
+    Throughout this guide, we use **People Chain** as the example. To set up a different parachain, substitute the chain spec file, snapshot path, and chain name with values for your target parachain.
+
+## Prerequisites
+
+### Hardware Requirements
+
+RPC nodes serving production traffic require robust hardware:
+
+- **CPU**: 8+ cores (16+ cores for high traffic)
+- **Memory**: 64 GB RAM minimum (128 GB recommended for high traffic)
+- **Storage**:
+    - Storage requirements vary by parachain. System parachains: Asset Hub (~600-800 GB), Bridge Hub (~500-600 GB), Collectives (~400-500 GB), People Chain (~300-400 GB), Coretime (~300-400 GB). For non-system parachains, check the [snapshot sizes](https://snapshots.polkadot.io/){target=\_blank} if available.
+    - Additional 200+ GB for relay chain pruned database
+    - Fast disk I/O is critical for query performance
+- **Network**:
+    - Public IP address
+    - 1 Gbps connection (for high traffic scenarios)
+    - Stable internet connection with sufficient bandwidth
+    - Open ports:
+        - 30333 (parachain P2P)
+        - 30334 (relay chain P2P)
+        - 9944 (Polkadot SDK WebSocket RPC)
+        - 9933 (Polkadot SDK HTTP RPC)
+        - 9615 (Prometheus metrics - optional)
```

Comment:
This should not be opened to the outside (e.g. on external firewall)

Reply:
Done.